src/components/Hub/HubNavigationBar.tsx

```diff
@@ -0,0 +1,76 @@
+import React, { ChangeEvent, useState } from "react"
+import { AppBar, InputBase, Tab, Tabs, Toolbar } from "@material-ui/core"
+import styled from "@emotion/styled"
+import SearchIcon from "@material-ui/icons/Search"
+
+function a11yProps(index: any) {
```

Comment:
Should index's type be `number` instead? It seems to be a `number` definitely.

Reply:
You are completely right!